src/core/readyException.js

```diff
@@ -0,0 +1,13 @@
+define( [
+	"../core"
+], function( jQuery ) {
+
+"use strict";
+
+jQuery.readyException = function( error ) {
```

Comment:
I think you understand it backwards - hook is not necessary, default handler very much is. 

I don't believe exceptions should be swallowed